### PR TITLE
Web scraping in r commit.

### PR DESCRIPTION
**Pull request by MasonCarhart**
An example of web scraping in R for LOST's other section.

### CODE REVIEW

**Comment by grantmcdermott** [comment]
Hi Mason,

Thanks for putting this up. Couple things, though: I'd prefer you to use a simple example example that doesn't require looping etc. to extract the elements. (My class example that you've use for inspiration here was meant for more advanced purposes.) With that in mind, could you please try something a little simpler? The rvest vignette has some nice examples, otherwise everyone likes to use IMBD pages.

**Comment by MasonCarhart** [comment]
Fixed a link, sorry (the syntax is new to me).  Also I can put a less complex example before this one for sure.  Or would you prefer me to put only the simple example up?

File: Other/simpl_web_scrape.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Implementation
+
+## R
+
+    ## Load and install the packages that we'll be using today
+    if (!require("pacman")) install.packages("pacman")
+
+    ## Loading required package: pacman
+
+    pacman::p_load(dplyr, rvest, data.table, hrbrthemes, stringi, XML)
```
Comment (grantmcdermott) [inline Other/simpl_web_scrape.md:32]: You don't need all of these packages. Just do `library(rvest)`.

File: Other/simpl_web_scrape.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Introduction
+
+  The “rvest” package is a web scraping package in R that provides a
+tremendous amount of versatility, as well as being easy to use. For this
+specific task, of web scrapping though pages on a website, we will be
```
Comment (grantmcdermott) [inline Other/simpl_web_scrape.md:5]: typos: should be "webscraping" and delete "though".

File: Other/simpl_web_scrape.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+  Also remember that web scraping is an art as much a science so play
+around with a problem and figure out creative ways to solve the issue,
+it might not pop out at you immediately.
+
+## Also Consider
+
+  Another useful function is html\_table(), if your data is able to be
+coerced into this type of object I would suggest trying this over the
+method bellow. Another good set of tools is html\_element and
```
Comment (grantmcdermott) [inline Other/simpl_web_scrape.md:20]: "below"

**Comment by grantmcdermott** [comment]
I've left a couple comment. the simple example is much better, thanks. Can you include one `html_table` example, though? A simple wiki table (e.g. [this one](https://simple.wikipedia.org/wiki/FIFA_World_Cup#Past_World_Cup_results)) would suffice.

**Comment by NickCH-K** [comment]
Is this one ready to go?

**Comment by MasonCarhart** [comment]
I would hope so, but Grant might think otherwise so I am hesitant to say yes.

**Comment by grantmcdermott** [comment]
I'll probably make a few changes to the intro to make it more generalizable, but I think we're good to go otherwise. Thanks @MasonCarhart.

**Pull request merged**